apps/challenges/aws_utils.py

```diff
@@ -311,205 +311,212 @@
     "executionRoleArn":"{EXECUTION_ROLE_ARN}",
     "networkMode":"awsvpc",
     "containerDefinitions":[
-        {{
-            "name": "{code_upload_container_name}",
-            "image": "{CODE_UPLOAD_WORKER_IMAGE}",
-            "essential": True,
-            "environment": [
-                {{
-                  "name": "AWS_DEFAULT_REGION",
-                  "value": "{AWS_REGION}"
-                }},
-                {{
-                  "name": "AWS_ACCESS_KEY_ID",
-                  "value": "{AWS_ACCESS_KEY_ID}"
-                }},
-                {{
-                  "name": "AWS_SECRET_ACCESS_KEY",
-                  "value": "{AWS_SECRET_ACCESS_KEY}"
-                }},
-                {{
-                  "name": "CLUSTER_NAME",
-                  "value": "{cluster_name}"
-                }},
-                {{
-                  "name": "CLUSTER_ENDPOINT",
-                  "value": "{cluster_endpoint}"
-                }},
-                {{
-                  "name": "CERTIFICATE",
-                  "value": "{certificate}"
-                }},
-                {{
-                  "name": "CIDR",
-                  "value": "{CIDR}"
-                }},
-                {{
-                  "name": "QUEUE_NAME",
-                  "value": "{queue_name}"
-                }},
-                {{
-                  "name": "EVALAI_API_SERVER",
-                  "value": "{EVALAI_API_SERVER}"
-                }},
+        {code_upload_container},
+        {submission_container}
+    ],
+    "requiresCompatibilities":[
+        "FARGATE"
+    ],
+    "cpu": "{CPU}",
+    "memory": "{MEMORY}",
+}}
+"""
 
-                {{
-                    "name": "AUTH_TOKEN",
-                    "value": "{auth_token}"
-                }},
+submission_container_defination = """
+{{
```

Comment:
@savish28 Also, we can't have have these huge task definitions as variables in `aws_utils.py`. Please move these to a separate file.

Reply:
@Ram81 Should we shift all task defn to a new file say `task_definations.py` and import all from there? or just these two container defns?

Reply:
Updated